week1/2-thursday/homework/exercise1/answer1.js

```diff
@@ -0,0 +1,9 @@
+///////// exercise 1 //////////
+
+function hello(name) {
+    // put your code here
+    return "Hello, " + name + '!';
+
+
+  }
+console.log(hello('Mustache'));
```

Comment:
I think this file is a duplicate.

Reply:
you're right, I see that I posted the answer onto both Thursday and Saturday twice

Reply:
BTW, when you want to remove a file that is already added to git, you would can use `git rm <filename>`.